docs/mergeduty/howto.md

```diff
@@ -86,11 +68,50 @@ After merge days, schedule a postmortem (or use current releaseduty postmortem)
 
 ### File tracking migration bug
 
-File a tracking migration bug if there isn't one. (e.g. [bug 1412962](https://bugzilla.mozilla.org/show_bug.cgi?id=1412962))
+File a tracking migration bug if there isn't one (e.g. [bug 1412962](https://bugzilla.mozilla.org/show_bug.cgi?id=1412962))
+
+### Run staging releases
+
+In order to prepare a smooth `b1` and `RC`, staging releases are to be run in the week before the mergeday day 1. In order for this to happen, we're using [staging releases submitted to try](https://firefox-source-docs.mozilla.org/tools/try/selectors/release.html).
+
+**For central to beta migration**
+
+- hop on `central` repository
+- make sure you're up to date with the tip of the repo
+- `mach try release --version <future-version-0b1> --migration central-to-beta`
+
+**For beta to release migration**
+
+- hop on `beta` repository
+- make sure you're up to date with the tip of the repo
+- `mach try release --version <future-version.0> --migration beta-to-release`
+
+These will create try pushes that look-alike the repos once they are merged. Once the decision tasks of the newly created CI graphs
+are green, staging releases can be created off of them via the [shipit-staging](https://shipit.staging.mozilla-releng.net/) instance.
+
+One caveat here is the list of partials that needs to be filled-in.
+:warning: The partials need to exist in S3 and be valid releases in [Balrog staging](https://balrog-admin.stage.mozaws.net/).
```

Comment:
It would be good to have a link to S3 here. Probably to http://ftp.stage.mozaws.net/pub/firefox/releases/